src/root.component.js

```diff
@@ -2,27 +2,13 @@ import React from "react";
 import { BrowserRouter, Route, Switch, Redirect } from "react-router-dom";
 import PeoplePage from "./people-page/people-page.component.js";
 
-export default class Root extends React.Component {
-  state = {
-    hasError: false
-  };
-
-  componentDidCatch(error, info) {
-    this.setState({ hasError: true });
-  }
-
-  render() {
-    return this.state.hasError ? (
-      <div className="mt-16">Error Loading people application</div>
-    ) : (
-      <div className="mt-16">
-        <BrowserRouter>
-          <Switch>
-            <Route path="/people/:personId" component={PeoplePage} />
-            <Route path="/people" component={PeoplePage} />
-          </Switch>
-        </BrowserRouter>
-      </div>
-    );
-  }
+export default function Root(props) {
+  return (
+    <div className="mt-16">
```

Comment:
On the master branch, the margin was wrong so that the people app was covered by the navbar. This fixes it.